src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -885,4 +885,19 @@ Do you want to continue?</value>
   <data name="Make_method_synchronous" xml:space="preserve">
     <value>Make method synchronous.</value>
   </data>
+  <data name="from_0" xml:space="preserve">
+    <value>from {0}</value>
+  </data>
+  <data name="Find_and_install_latest_version" xml:space="preserve">
+    <value>Find and install latest version</value>
```

Comment:
I see where the name-less versions are used above, but why don't we want to show the package name in that case?